src/manage/manage-bar.ts

```diff
@@ -32,11 +49,27 @@ export class ManageBar extends LitElement {
    */
   @property({ type: Boolean }) showUnselectAll = false;
 
+  /**
+   * Whether item has deleted or not (default true)
+   */
+  @property({ type: Boolean }) hasItemsDeleted = true;
```

Comment:
I'm curious about how this is named & used. It seems incorrect that the default `true` implies items have already been deleted before anything happens. This is essentially just a switch for whether to show the error dialog, right?

Maybe this would be clearer in both ways if we inverted the logic, e.g., `hasRemoveItemsError = false;`

Alternatively, could we just call a method to pop up the error dialog when needed? I think it would be reasonable to handle opening the dialog via a method call rather than a reactive property, especially since there is no corresponding way to close the dialog via this component.